src/api/routes.py

```diff
@@ -121,4 +121,58 @@ def get_one_product(id):
     if one_product:
         return jsonify(one_product.to_dict()), 200
     
-    return({"error": "Product not found"}), 404
\ No newline at end of file
+    return({"error": "Product not found"}), 404
+
+    @api.route('/client/<int:id>/favorites', methods=['POST'])
+@jwt_required()
+def add_whish(id):
+    current_user = get_jwt_identity()
+    if current_user["id"] != id:
+        return {'error': 'Invalid action'}, 400
+
+    id_product = request.json.get("have_product", None)
+
+    new_whish = Product(
+        from_account = current_user["id"],
+        have_product =  id_product,
+    )
+
+    try:
+        new_whish.create()
+        return jsonify(new_whish.to_dict())
+    except exc.IntegrityError: 
+        return {"error":"something went wrong"}, 409
+
+@api.route('/client/<int:id>/favorites', methods=['DELETE'])
+@jwt_required()
+def remove_wish(id):
+    user_with_wish = get_jwt_identity()
+    if user_with_wish["id"] != id:
```

Comment:
en python es mejor usar not que !

Reply:
Bastante cierto, los acentos los andaluces lo llevamos mal ya sabes y el de javascript es fuerte en mi.